src/pydocstyle/checker.py

```diff
@@ -921,4 +926,4 @@ def get_function_args(function_string):
     function_arg_node = ast.parse(textwrap.dedent(function_string)).body[0].args
     arg_nodes = function_arg_node.args
     kwonly_arg_nodes = function_arg_node.kwonlyargs
-    return set(arg_node.arg for arg_node in chain(arg_nodes, kwonly_arg_nodes))
+    return list(arg_node.arg for arg_node in chain(arg_nodes, kwonly_arg_nodes))
```

Comment:
Just use list comprehension instead of calling `list` on a generator expression: `return [arg_node.arg for ...]`.

Reply:
done :)